### PR TITLE
docs: simplify install docs

### PR DESCRIPTION
**Pull request by bpmct**
While we support a diverse amount of installation methods, only power users will want to use a system service/non-install script. Folks who want to run a system service on a full-time VM should understand how to start it and keep it running. We can always make a guide/tutorial for this that walks through the basics

| Before | After |
| -- | -- |
| Screenshot 2024-01-30 at 4 18 30 PM| image |

Main concerns with the old one:
- Nested tabs
- Too many decisions for a novice user

### CODE REVIEW

File: docs/install/index.md [hunk 1]
```diff
@@ -1,251 +1,47 @@
...
-
-</div>
+Refer to [GitHub releases](https://github.com/coder/coder/releases) for alternate installation methods (e.g. standalone binaries, system packages).
 
 ## Windows
 
-<div class="tabs">
-
-## Winget
+Use [GitHub releases](https://github.com/coder/coder/releases) to download the Windows installer (`.msi`) or standalone binary (`.exe`).
```
Comment (matifali) [inline docs/install/index.md:19]: Why preferring installer over Winget while Winget is built-in to Windows and also supports arm64 out of the box?

**Review by matifali — COMMENTED** [review]
I am ok with this refactoring but we are now burying all the installation m this we support. We should have them somewhere got the people who need. 

Probably the full original detailed installation methods could be moved to a tutorial or guide in the guides section. 
I don't think we should just delete it.

**Comment by matifali** [comment]
Could you copy the existing install sh as is to a new guide with a title "How to install Coder?"

**Review by matifali — CHANGES_REQUESTED** [review]
I am ok with this refactoring but we are now removing all the other installation methods we support. We should have them somewhere for the people who need. 

Probably the full original detailed installation methods could be moved to a tutorial or guide in the guides section. 
I don't think we should just delete it. 

Could you copy the existing install sh as is to a new guide with a title "How to install Coder?"

File: docs/install/index.md [hunk 2]
```diff
@@ -1,251 +1,47 @@
...
+```sh
+coder login https://coder.example.com
+```
 
-## Next steps
+## Next up
 
-- [Configuring Coder](../admin/configure.md)
-- [Templates](../templates/index.md)
+- [Create your first template](./1-click.md)
```
Comment (matifali) [inline docs/install/index.md:47]:
```suggestion
- [Create your first template](../templates/index.md)
```

File: docs/install/index.md [hunk 3]
```diff
@@ -1,251 +1,47 @@
...
-
-You can modify the installation process by including flags. Run the help command
-for reference:
-
-```shell
-curl -fsSL https://coder.com/install.sh | sh -s -- --help
-```
-
-</div>
+Refer to [GitHub releases](https://github.com/coder/coder/releases) for alternate installation methods (e.g. standalone binaries, system packages).
```
Comment (matifali) [inline docs/install/index.md:15]:
```suggestion
Alternatively, you can use the [Homebrew](https://brew.sh/) package
manager to install Coder.

	```shell
	brew install coder/coder/coder
	```

Refer to [GitHub releases](https://github.com/coder/coder/releases) for alternate installation methods (e.g. standalone binaries, system packages).
```

Do we want to mention our Homebrew packages anywhere?

**Review by matifali — CHANGES_REQUESTED** [review]

**Review by matifali — APPROVED** [review]
I revisited and agree with the refactoring. A few suggestions regarding wrong/old links and it's good to merge.

**Comment by matifali** [comment]
@bpmct, do you want to make any further changes before merging?

**Comment by bpmct** [comment]
Yeah, I have several changes planned! Should have the rest pushed soon

**Comment by matifali** [comment]
@bpmct I think you still want to complete this.

**Pull request merged**